src/App.js

```diff
@@ -6,13 +6,16 @@ import Login from "./components/login"
 import Feed from "./components/feed"
 import Editor from "./components/editor"
 import Films from "./components/films"
-import Profile from "./components/profilePage";
+import Settings from "./components/settings";
 import Navbar from "./components/navbar"
+import firebase from "firebase";
+
 
 class App extends Component {
   state = {
     signedIn: false,
-    googleId: ''
+    googleId: '',
+    profilePic: ''
```

Comment:
do we need profilepic in app?

Reply:
not anymore, i'll remove it

Reply:
also removed the line of code where we set the state for profile in app.js